tests/shared/importers/test_importer.py

```diff
@@ -190,14 +190,14 @@ async def test_import_nlu_training_data_from_e2e_stories(
             StoryStep(
                 events=[
                     SlotSet("some slot", "doesn't matter"),
-                    UserUttered("greet_from_stories", {"name": "greet_from_stories"}),
+                    UserUttered(intent={"name": "greet_from_stories"}),
```

Comment:
optional: would be great to have on example with entities (can also be done in the `e2e` PR)